lib/global-styles.php

```diff
@@ -686,6 +693,7 @@ function gutenberg_experimental_global_styles_normalize_schema( $tree ) {
 		),
 		'settings' => array(
 			'color'      => array(),
+			'custom'     => array(),
```

Comment:
The CSS variables are like styles e.g: one can do:
```
:root {
	--wp-test-var: 4;
	background-color: black;
}
```

Given that css variables are like styles wouldn't it make sense to set the variables together with the styles?

Reply:
Presets are also styles, though. I think it makes sense having presets & this custom thing together as they behave the same: name is generated based on the object shape, both generate CSS vars (so aren't attached to any style prop).